apps/vanilla/src/pages/index.html

```diff
@@ -7,104 +7,106 @@
     <title>Vite App</title>
   </head>
   <body class="container">
-    <header class="header font-logo">📚 Books app</header>
+    <header class="header font-header">
+      <div>📚 Books app</h1>
```

Comment:
Unmatch start and end tag.
```suggestion
      <h1>📚 Books app</h1>
```